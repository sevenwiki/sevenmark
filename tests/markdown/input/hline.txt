Text before horizontal line.
-----
Text after horizontal line.