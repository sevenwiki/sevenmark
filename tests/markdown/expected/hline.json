[
  {
    "Text": {
      "location": {
        "start": 0,
        "end": 28
      },
      "content": "Text before horizontal line."
    }
  },
  {
    "Text": {
      "location": {
        "start": 28,
        "end": 29
      },
      "content": "\n"
    }
  },
  "HLine",
  {
    "Text": {
      "location": {
        "start": 35,
        "end": 62
      },
      "content": "Text after horizontal line."
    }
  }
]